pom.xml

```diff
@@ -157,6 +157,12 @@
                 <artifactId>org.wso2.carbon.identity.oauth</artifactId>
                 <version>${org.wso2.carbon.identity.oauth.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.wso2.carbon.identity.inbound.auth.oauth2</groupId>
+                <artifactId>org.wso2.carbon.identity.oauth.dcr</artifactId>
+                <version>${org.wso2.carbon.identity.oauth.version}</version>
+                <scope>provided</scope>
```

Comment:
why we need the scope as provided?

Reply:
Fixed with [0b159f1](https://github.com/wso2-extensions/identity-carbon-auth-rest/pull/269/commits/0b159f19942e188fe8ce8e4c427748c3bc23e08c)